test/tests.js

```diff
@@ -6828,7 +6827,7 @@ describe("Tables", function () {
                 [c3, c4]
             ]);
             var result = table._iterateLayout(500, 500);
-            verifyLayoutResult(result, [0, 0], [220, 220], [480, 20], [50, 10], true, false, "");
+            verifyLayoutResult(result, [5, 5], [225, 225], [490, 0], [50, 0], false, false, "");
```

Comment:
Can you move this comment to the `.ts` file?  Trying to get context in a `.js` file is a little difficult